LICENSE

```diff
@@ -2,15 +2,15 @@
   LICENSE ISSUES
   ==============
 
-  The OpenSSL toolkit stays under a dual license, i.e. both the conditions of
+  The OpenSSL toolkit stays under a double license, i.e. both the conditions of
```

Comment:
Not sure I agree.  "dual license" is a common terminology that's been used for years and years